addons/binding/org.openhab.binding.kodi/README.md

```diff
@@ -0,0 +1,64 @@
+# Kodi Binding
+
+This binding integrates the Kodi players (which used to be XBMC).
+
+## Introduction
+
+
+## Supported Things
+
+This binding supports only one thing: The Kodi player. All kodi devices are registered as an audio sink in the framework.
+
+
+## Discovery
+
+This binding can discover the Kodi players. 
+
+## Binding Configuration
+
+The binding can auto-discover the Kodi players present on your local network. The auto-discovery is enabled by default. To disable it, you can create a file in the services directory called Kodi.cfg with the following content:
+
+```
+org.openhab.Kodi:enableAutoDiscovery=false
+```
+
+This configuration parameter only controls the Kodi player auto-discovery process, not the openHAB auto-discovery. Moreover, if the openHAB auto-discovery is disabled, the Kodi player auto-discovery is disabled too.
+
+The binding has the following configuration option, which can be set for "binding:kodi":
+
+| Parameter | Name    | Description  | Required |
+|-----------------|------------------------|--------------|------------ |
+| callbackUrl | Callback URL | URL to use for playing notification sounds, e.g. http://192.168.0.2:8080 | no |
+
+
+## Thing Configuration
+
+The Kodi player thing requires the ip address and the port to access it on. 
+In the thing file, this looks e.g. like
+```
```

Comment:
please add an empty line above code sections as otherwise Jekyll breaks the layout.